skimage/measure/profile.py

```diff
@@ -27,6 +27,10 @@ def profile_line(image, src, dst, linewidth=1,
         How to compute any values falling outside of the image.
     cval : float, optional
         If `mode` is 'constant', what constant value to use outside the image.
+    reduce_func : NumPy ufunc, default np.mean
```

Comment:
Not necessarily a Numpy ufunc, it can be any `ufunc` right?

Reply:
I'd go with `reduce_func : callable, optional` and explain the rest inside the description. If we use @jni's try except suggestion we can essentially use any callable that takes an array and returns a value or array.